AppPrawject/AppPrawject/Views/Pet/Index.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@* Model definition is done here*@
+@model List<Pet>
+
+<h1>This is the list of Pets</h1>
+
+
+
+<table class="table">
+    <tr>
+        <th>Name</th>
+        <th>Breed</th>
+        <th>Weight</th>
+        <th>Image</th>
+    </tr>
+
+
+    @if (Model.Count == 0)
+    {
+
+        <tr>
+            <td colspan="4">
+                You do not have pets listed. <a href="/pet/add">Add your pet here.</a>
+            </td>
+        </tr>
+    }
+
+
+
+    @foreach (var pet in Model)
+    {
+        <tr>
+            <td>@pet.Name</td>
+            <td>@pet.Breed</td>
+            <td>@pet.Weight</td>
+            <td>@pet.Image</td>
+
+        </tr>
+    }
+</table>
+
+@if (Model.Count > 0)
```

Comment:
I would recommend to move this button before your `<table>` - that way if the table gets to big the user won't have to scroll in order to find the add button all the way at the bottom of the page